### PR TITLE
Css basics

### PR DESCRIPTION
**Pull request by syboulette**

### CODE REVIEW

File: app/assets/stylesheets/config/_fonts.scss [hunk 1]
```diff
@@ -1,10 +1,44 @@
...
+$headers-font: "area-normal", sans-serif, 40px;
+
+
+h1 {
+  font-family: "area-normal", sans-serif;
+font-weight: 700;
+font-style: normal;
+text-align: center;
+font-size: 40px;
+
```
Comment (akramerin) [inline app/assets/stylesheets/config/_fonts.scss:17]: there is just some styling issues - please adjust indentation and delete the empty line

File: app/assets/stylesheets/config/_fonts.scss [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+}
+h2 {
+  font-family: "area-normal", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-align: left;
+  font-size: 32px;
+  text-decoration: underline;
```
Comment (akramerin) [inline app/assets/stylesheets/config/_fonts.scss:25]: are we sure about the underline? personally i think this is a little bit outdated for a headline and prefer it without it, but i guess it is a personal preference

**Review by akramerin — COMMENTED** [review]
Just added two comments (one styling issure, one frontend preferences) but nothing major.

File: app/assets/stylesheets/config/_fonts.scss (hunk 2 above)
Comment (D-Zaleski) [reply app/assets/stylesheets/config/_fonts.scss:25]: I would agree with Ariane on this one but would have to see in effect to know for sure, so leave it as is for now.

**Review by D-Zaleski — APPROVED** [review]
lgtm. We can always change these settings in the future.

**Pull request merged**